portal/src/main/webapp/js/src/crosscancer.js

```diff
@@ -85,28 +85,64 @@
 
         var isThereHetLoss = false;
         var isThereGain = false;
-        var filterAndSortData = function(histDataOrg) {
+        var filterAndSortData = function(histDataOrg, sliderValue, metaData, totalSamSliderValue) {
+            var threshold = 0;
+            if(sliderValue !== undefined && sliderValue !== null){
```

Comment:
@zhx828 Thank you very much for the comment about non number input.
If it's a non-number input, the slider will change that to NaN but  _.isNumber() won't be able to catch that since it returns true if object is a Number (including NaN).
So I changed as below, let me know if any thoughts.
!_.isNaN(totalSamSliderValue) && !_.isUndefined(totalSamSliderValue)